administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-08-13.sql

```diff
@@ -0,0 +1,8 @@
+-- after 4.0.0 RC6
+UPDATE "#__assets"
+SET "name" = 'com_content.stage.1'
+WHERE "name" = 'com_content.state.1';
+
+UPDATE "#__workflow_stages"
+SET "asset_id" = 57
```

Comment:
```suggestion
SET "asset_id" = 57
```
See my review comment for the same file for MySQL above: We cannot use hard-coded asset IDs on updated sites.

Reply:
So I suggest:

```
UPDATE "#__workflow_stages"
   SET "asset_id" = (SELECT MAX("id") FROM "#__assets" WHERE "name" = 'com_content.stage.1');
```